translations/ui-erm-usage/en.json

```diff
@@ -326,16 +326,19 @@
   "statistics.multi.delete.confirmation": "The selected <strong>{count} {count, plural, one {report} other {reports}}</strong> will be <strong>deleted</strong>.",
   "statistics.multi.delete.infoText": "Overview about harvested statistics per year. Expand a year and click a colored button to download/delete report or get additional info. Select checkboxes to select reports for deletion.",
 
-  "permission.module.enabled": "UI: eUsage module is enabled",
-  "permission.generalSettings.manage": "Settings (eUsage): Can view and edit settings",
+  "permission.module.erm-usage.enabled": "UI: eUsage module is enabled",
+  "permission.settings.erm-usage.enabled": "Settings (eUsage): Display list of settings pages",
+  "permission.generalSettings.manage": "Settings (eUsage): Can view and edit all settings",
   "permission.all": "eUsage: All permissions",
-  "permission.view": "eUsage: Can view usage data providers and usage reports",
-  "permission.view-create": "eUsage: Can view and create usage data providers and usage reports",
-  "permission.view-edit": "eUsage: Can view and edit usage data providers and usage reports",
-  "permission.view-create-edit": "eUsage: Can view, create and edit usage data providers and usage reports",
-  "permission.view-create-edit-delete": "eUsage: Can view, create, edit and delete usage data providers and usage reports",
-  "permission.harvester.all": "eUsage harvester: All permissions for eUsage harvester module",
-  "permission.harvester.view": "eUsage harvester: Read-only permissions for eUsage harvester module",
+  "permission.view": "eUsage: Can view usage data providers and view/download usage reports",
+  "permission.udp.create": "eUsage: Can create usage data providers",
+  "permission.reports.create": "eUsage: Can upload usage reports",
+  "permission.udp.edit": "eUsage: Can edit usage data providers",
+  "permission.udp.create-edit": "eUsage: Can create and edit usage data providers",
+  "permission.udp.delete": "eUsage: Can delete usage data providers",
+  "permission.reports.delete": "eUsage: Can delete usage reports",
```

Comment:
Not sure about 
_"eUsage: Can delete usage reports"_ and _"eUsage: Can upload usage reports"_
Maybe its better using just _"reports"_ (without usage) or _"usage data providers reports"_?

Reply:
This PR focuses on aligning the permission translations with the definitions in `package.json`, rather than addressing permission names or descriptions.

Reply:
Sure, I understand that.
But anyway, what is a "usage report"? This expression is appearing here for the first time. 
Maybe we should not start using new expressions just for the permissions.

Reply:
This PR is not introducing any new expressions; it is strictly focused on updating translations to align with the expressions that were defined over two years ago based on a Product Owner decision. If there are concerns about how permissions are described, please discuss them directly with the PO.

See https://folio-org.atlassian.net/browse/UIEUS-292 / https://github.com/folio-org/ui-erm-usage/commit/4cb502b4c795efdb70731c1f4fbf1eec81a02e3e

Reply:
Maybe its talk with Stefan Dombek about that. So he should know about that too.